Snowplow/SPSession.m

```diff
@@ -123,8 +123,12 @@ - (void) setCheckInterval:(NSInteger)checkInterval {
     [self startChecker];
 }
 
-- (SPPayload *) getSessionDict {
+- (NSMutableDictionary *) getSessionDictWithEventId:(NSString *)firstEventId {
     [self updateAccessedLast];
+    //if (_firstEventId == nil) {
+    //    _firstEventId = firstEventId;
+    //    [self addFirstEventIdToDict];
+    //}
```

Comment:
This is to remove conflict with schemas, my fix was to remove all conflicts in one commit that I can later delete.